src/python/turicreate/toolkits/_model.py

```diff
@@ -649,6 +653,7 @@ def _name(self):
         --------
         >>> model_name = m.name()
         """
+        warnings.warn("CustomModel.name() is deprecated. It will be removed in the next major release. Use type(CustomModel) instead.")
```

Comment:
Change the message to. 'This function is deprecated. It will be removed in the next release.'

Reply:
Good!